user-access-handlers/src/test/java/no/unit/nva/handlers/models/CreateUserRequestTest.java

```diff
@@ -1,31 +0,0 @@
-package no.unit.nva.handlers.models;
-
-import static no.unit.nva.testutils.RandomDataGenerator.randomString;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.core.Is.is;
-import static org.hamcrest.core.IsEqual.equalTo;
-import static org.hamcrest.core.IsInstanceOf.instanceOf;
-import java.io.IOException;
-import no.unit.nva.identityservice.json.JsonConfig;
-import org.junit.jupiter.api.Test;
-
-class CreateUserRequestTest {
```

Comment:
No needed longer, this class has been testing deserialization of json alias - alias is removed now